plugins/pencilblue/controllers/admin/plugins/plugin_settings.js

```diff
@@ -128,6 +135,36 @@ module.exports = function(pb) {
             });
         });
     };
+    function insertGroupHeadings(clone){
+        var previous = clone[clone.length - 1], current;
+        for (var i = clone.length - 1; i >= 0; i--) {
+            current = clone[i];
+            if (current.heading != previous.heading) {
+                clone.splice(i+1, 0, {id: previous.heading, isHeading: true, heading: convertNameToDisplayName(previous.heading)});
+            }
+
+            previous = current;
+        }
+    }
+    function sortOn(property){
```

Comment:
Starting in 0.7.0 we are going to bump the minimum node version to 4.x.  We are doing this in order to provide support for ES6 classes.  We'll slowly be making that transition over the course of the next year or so.  In support of that we ask that all functions are declared as part of the prototype, ```PluginSettingsFormController.sortOn = function(property) {```.  This will make it easier for us to convert as well as keep functions public so that they can be unit tested.